7.1/Dockerfile

```diff
@@ -3,12 +3,12 @@ FROM openshift/base-centos7
 # This image provides an Apache+PHP environment for running PHP
 # applications.
 
-MAINTAINER Diego Hernandes <diego@hernandev.com>
```

Comment:
In this file I renamed 7.0 to be 7.1 as I had not really thought about sending a PR. Looking at http://php.net/supported-versions.php 7.0 is out of active support with security patching ending in 9 months to dropping support now so that people move to 7.1 probably isn't such a bad thing.